tests/e2e/common-actions/databases-actions.ts

```diff
@@ -0,0 +1,31 @@
+import { t } from 'testcafe';
+import { MyRedisDatabasePage } from '../pageObjects';
+
+const myRedisDatabasePage = new MyRedisDatabasePage();
+
+export class DatabasesActions {
+    /**
+     * Verify that databases are displayed
+     * @param databases The list of databases to verify
+     */
+    async verifyDatabasesDisplayed(databases: string[]): Promise<void> {
+        for (const db of databases) {
+            const databaseName = myRedisDatabasePage.dbNameList.withText(db);
+            await t.expect(databaseName.exists).ok(`"${db}" database doesn't exist`);
+        }
+    }
+
+    /**
+     * Import database using file
+     * @param pathToFile The path to file for import
+     * @param databaseType The database type
```

Comment:
What is that? why not creating a type if there are several options?

Reply:
thanks, created ImportDatabaseParameters type